clang/lib/Driver/Driver.cpp

```diff
@@ -4482,55 +4625,77 @@ void Driver::BuildActions(Compilation &C, DerivedArgList &Args,
       UnbundlerInputs.push_back(LI);
     }
     const Arg *LastArg;
-    auto addUnbundlerInput = [&](types::ID T, const Arg *A) {
+    auto addUnbundlerInput = [&](types::ID T, const StringRef &A) {
```

Comment:
```suggestion
    auto addUnbundlerInput = [&](types::ID T, StringRef A) {
```
Since it already represents a constref (https://llvm.org/doxygen/classllvm_1_1StringRef.html). If suitable, the same applies to `addUnbundlerArgs()` and a lambda within `getLinkerArgs()`

Reply:
I did some quick tinkering here and found I was able to modify `A` within the lambda when removing the constref in the parameter.

Reply:
I see, thanks! I'll go read up on `StringRef` more :)